tests/laplace.hpp

```diff
@@ -53,7 +53,7 @@ class Laplace
   void run(PreconditionerType &preconditioner,
            dealii::Function<dim> const &source);
 
-private:
+  // private:
```

Comment:
We should just add a comment saying that this should be private but it's not for simplicity

Reply:
Agreed.